radio/src/gui/128x64/radio_setup.cpp

```diff
@@ -188,7 +181,7 @@ void menuRadioSetup(event_t event)
     CASE_IMU(LABEL(IMU))
     CASE_IMU(0)
     CASE_IMU(0)
-    LABEL(ALARMS), 0, CASE_CAPACITY(0)
```

Comment:
Shouldn't the target section be deleted too then ?

#if defined(TX_CAPACITY_MEASUREMENT)
      case ITEM_RADIO_SETUP_CAPACITY_WARNING:
        lcdDrawTextAlignedLeft(y, STR_CAPAWARNING);
        drawValueWithUnit(LCD_W-2, y, g_eeGeneral.mAhWarn*50, UNIT_MAH, attr|RIGHT) ;
        if(attr) CHECK_INCDEC_GENVAR(event, g_eeGeneral.mAhWarn, 0, 100);
        break;
#endif

Reply:
Thanks. I've removed this code plus associated code from a few other files. Also removed unused strings only used by this code.